.github/workflows/deploy.yml

```diff
@@ -22,50 +22,54 @@ jobs:
       Application: ${{ github.event.repository.name }}
     secrets: inherit
 
-  qa-via:
+  QA:
     needs: create-image
-    name: ${{ github.event.repository.name }}
-    uses: hypothesis/workflows/.github/workflows/eb-update.yml@main
+    uses: hypothesis/workflows/.github/workflows/deploy.yml@main
     with:
-      Application: ${{ github.event.repository.name }}
-      Environment: qa
-      Region: us-west-1
-      Operation: deploy
-      Version: latest
+      operation: deploy
+      github_environment_name: QA
+      github_environment_url: https://qa-via.hypothes.is/
+      aws_region: us-west-1
+      elasticbeanstalk_application: via
+      elasticbeanstalk_environment: qa
+      version: ${{ needs.create-image.outputs.docker_tag }}
     secrets: inherit
 
-  qa-lms-via:
+  QA (Edu):
     needs: create-image
-    name: lms-via
-    uses: hypothesis/workflows/.github/workflows/eb-update.yml@main
+    uses: hypothesis/workflows/.github/workflows/deploy.yml@main
     with:
-      Application: lms-via
-      Environment: qa
-      Region: us-west-1
-      Operation: deploy
-      Version: latest
+      operation: deploy
+      github_environment_name: QA (Edu)
```

Comment:
I'm gonna call these `QA (Edu)` and `Production (Edu)` instead of `QA (LMS)` and `Production (LMS)`. It's confusing to see the name of one app (`LMS`) in the name of an environment belonging to another app (`Via`). When you see something like `Via / Production (LMS)` it's unclear whether you're looking at the name of a Via environment or an LMS environment since both names Via and LMS are in the environment name.